### ISSUE TITLE
Javascript errors on loading page

### ISSUE THREAD

**Issue by akariv** [openspending/wheredoesmymoneygo.org#14]
Includes one deprecation warning (due Sep-2018) and some resource loading errors (apparently it tries to load some fonts from openspending.org?)

<img width="855" alt="screen shot 2018-08-28 at 22 19 55" src="https://user-images.githubusercontent.com/265333/44745513-9b695c80-ab10-11e8-878b-61fffc43d3aa.png">

/cc @rufuspollock

**Comment by rufuspollock** [comment]
So the real error looks to be the failure to load the bubblemap.js stuff stuff from legacy due to an invalid cert. Can we fix that?

**Comment by akariv** [comment]
Yes, you're right - after accepting the certificate as valid the website seems to work.

**Comment by brew** [comment]
I've added an SSL cert to https://legacy.openspending.org and fixed some links within WDMMG to use the correct API endpoint, and have assets load with https. This has fixed up some pages, but pages that call the search api at https://legacy.openspending.org return errors.

- https://app.wheredoesmymoneygo.org/dailybread.html : Although this page loads, a call to the domain http://taxman.openspending.org fails because the subdomain doesn't exist.
- https://app.wheredoesmymoneygo.org/bubbletree-map.html : This loads, and appears functional.
- https://app.wheredoesmymoneygo.org/spending.html : Makes a call to the search api which is broken, so this page is broken.
